addons/website/views/snippets/s_images_constellation.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_images_constellation" name="Images constellation">
+    <section class="s_images_constellation o_cc o_cc5 pt48 pb48">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="23">
+                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-4 col-lg-3 d-none d-lg-block o_snippet_mobile_invisible" style="z-index: 1; grid-area: 3 / 6 / 7 / 9; --grid-item-padding-x: 0px; --grid-item-padding-y: 0px;">
+                    <img class="img img-fluid mx-auto rounded" src="/web/image/website.s_carousel_default_image_2" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-4 g-height-6 col-12 col-lg-4" style="z-index: 2; grid-area: 7 / 1 / 13 / 5; --grid-item-padding-x: 16px; --grid-item-padding-y: 0px;">
+                    <img class="img img-fluid mx-auto rounded" src="/web/image/website.s_picture_default_image" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-4 g-height-6 col-lg-4 d-none d-lg-block o_snippet_mobile_invisible" style="z-index: 3; grid-area: 7 / 9 / 13 / 13; --grid-item-padding-x: 0px; --grid-item-padding-y: 24px;">
+                    <img class="img img-fluid mx-auto rounded" alt=""  data-shape="web_editor/geometric/geo_sonar" data-file-name="s_images_constellation_default_image.jpg" data-original-mimetype="image/jpeg/" src="/web_editor/image_shape/website.s_images_constellation_default_image/web_editor/geometric/geo_sonar.svg"/>
+                </div>
+                <div class="o_grid_item g-height-8 g-col-lg-6 col-12 col-lg-6 justify-content-center" style="z-index: 6; grid-area: 9 / 5 / 17 / 11; --grid-item-padding-x: 16px; --grid-item-padding-y: 16px;">
```

Comment:
Why did you add `justify-content-center` ? I don't think the editor can add this class, and it seems unnecessary.

Reply:
left-over indeed 👍